content/en/docs/instrumentation/java/manual.md

```diff
@@ -55,93 +271,277 @@ telemetry source-identifying info.
+auto-configuration extension through the
+`opentelemetry-sdk-extension-autoconfigure` module. It is recommended to use
+auto-configuration, as it is easier to use and comes with various additional
+capabilities.
+
+#### Automatic Configuration
+
+To use auto-configuration add the following dependency to your application:
 
-### Gradle
+{{< tabpane text=true >}} {{% tab Gradle %}}
 
 ```kotlin
 dependencies {
-    implementation 'io.opentelemetry:opentelemetry-api:{{% param javaVersion %}}'
-    implementation 'io.opentelemetry:opentelemetry-sdk:{{% param javaVersion %}}'
-    implementation 'io.opentelemetry:opentelemetry-exporter-otlp:{{% param javaVersion %}}'
-    implementation 'io.opentelemetry:opentelemetry-semconv:{{% param javaVersion %}}-alpha'
+    implementation 'opentelemetry-sdk-extension-autoconfigure:{{% param javaVersion %}}'
+    implementation 'opentelemetry-sdk-extension-autoconfigure-spi:{{% param javaVersion %}}'
 }
 ```
 
-See [releases][releases] for a full list of artifact coordinates.
+{{% /tab %}} {{% tab Maven %}}
+
+```xml
+<project>
+    <dependencies>
+        <dependency>
+          <groupId>io.opentelemetry</groupId>
+          <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>io.opentelemetry</groupId>
+          <artifactId>opentelemetry-sdk-extension-autoconfigure-spi</artifactId>
+        </dependency>
+    </dependencies>
+</project>
+```
+
+{{% /tab %}} {{< /tabpane>}}
+
+It allows you to auto-configure the OpenTelemetry SDK based on a standard set of
+supported environment variables and system properties. Each environment variable
+has a corresponding system property named the same way but as lower case and
+using the `.` (dot) character instead of the `_` (underscore) as separator.
+
+The logical service name can be specified via the `OTEL_SERVICE_NAME`
+environment variable (or `otel.service.name` system property).
+
+The traces, metrics or logs exporters can be set via the `OTEL_TRACES_EXPORTER`,
+`OTEL_METRICS_EXPORTER` and `OTEL_LOGS_EXPORTER` environment variables. For
+example `OTEL_TRACES_EXPORTER=logging` configures your application to use an
+exporter that writes all traces to the console. The corresponding exporter
+library has to be provided in the classpath of the application as well.
+
+For debugging and local development purposes, use the `logging` exporter. After
+you have finished setting up manual instrumentation, provide an appropriate
+exporter library in the classpath of the application to
+[export the app's telemetry data](/docs/instrumentation/java/exporters/) to one
+or more telemetry backends.
+
+The SDK auto-configuration has to be initialized from your code in order to
+allow the module to go through the provided environment variables (or system
+properties) and set up the `OpenTelemetry` instance by using the builders
+internally.
```

Comment:
I think it's important to emphasize _where_ in the code as well. The example shows where, but explaining more generally where you'd do it matters.

Reply:
As early as possible in the application lifecycle.

Reply:
fair point, it's missing some "as early as possible" or "ideally in your main method" etc.